### PR TITLE
Add aws scenario, create a kops-aws-canary job

### PR DESCRIPTION
**Pull request by krzyzacy**
Test kops-aws-e2e out.

It's more like aws-e2e can roll into the regular e2e scenario since it has less unique stuff.

Haven't run it locally yet ( too lazy to set up aws stuff, instead I'll ask jenkins do the job for me ).

### CODE REVIEW

**Comment by krzyzacy** [comment]
@zmerlynn @justinsb just put up something together, please review/comment and I'll iterate on the PR.

**Comment by zmerlynn** [comment]
Let's wait until Tuesday morning (when I'm back in the office) to get it in, unless you want to iterate further.

**Comment by krzyzacy** [comment]
@zmerlynn thanks! Sounds good let's wait till then.

File: scenarios/kubernetes_aws.py [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+        raise ValueError('HOME dir not set!')
+
+    PARSER.add_argument(
+        '--aws-ssh',
+        default='%s/.ssh/kube_aws_rsa' % HOME,
+        help='Path to .ssh/google_compute_engine keys')
+    PARSER.add_argument(
+        '--aws-pub',
+        default='%s/.ssh/kube_aws_rsa.pub' % HOME,
+        help='Path to pub gce ssh key')
```
Comment (krzyzacy) [inline scenarios/kubernetes_aws.py]: yup merit of copy-pasting

**Comment by krzyzacy** [comment]
rebased, will merge on green :-)

**Comment by krzyzacy** [comment]
Apply the label based on the approval

**Pull request merged**